ultraLatency.py

```diff
@@ -198,6 +198,7 @@
         oh.setTriggerSource(ohboard,options.gtx,0x5) # GBT, 0x0 for GTX
         pass
 
+    oh.setTriggerThrottle(ohboard, options.gtx, options.throttle)
```

Comment:
On the contrary, how do you know what this register is set to when you launch another scan.  Say you launch a scan with throttle 500, then you launch another scan with throttle 0.  In between scans throttle is never overwritten so your next scan with throttle 0 is run with throttle 500.

confChamber right now is not changing this method (maybe it should, but I would prefer not since it seems to only be relevant for latency scans).

I think we should follow the statement of Misha above.  Leave it as it is, and ensure it's written each time but make the default value assigned (0) correspond to the unthrottled case.

If we notice this causes instabilities I would be happy then to submit a new issue on the repo and resolve the bug fix but I think this implementation is preferred.  You know the state of the system when you start.